src/egon/data/datasets/power_plants/pv_rooftop.py

```diff
@@ -160,7 +160,13 @@ def pv_rooftop_per_mv_grid_and_scenario(scenario, level):
             WHERE carrier = 'solar_rooftop'
             AND scenario_name = '{scenario}'
             """
-        ).capacity[0]
+        )
+
+        if target.empty:
+            print(f"No PV rooftop in scenario {scenario}")
```

Comment:
I think this is necessary because we are getting a unique solar carrier for both solar ground-mounted and rooftop. Do you know if that will be corrected in the next models coming from Pypsa-eur?